LadybugTools_oM/Versioning_71.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Property": {
+    "ToNew": {
+      "BH.oM.LadybugTools.RunSimulationCommand.EpwFile": "EPWFile"
+    },
+    "ToOld": {
+      "BH.oM.LadybugTools.RunSimulationCommand.EPWFile": "EpwFile"
```

Comment:
I think this needs to be the full property names for the new versions to pass versioning?

Reply:
It passed versioning already 😄 - every time I do property updates I forget how to do it and re-read the docs